src/main/java/com/g4s8/ghman/bot/TkNotifications.java

```diff
@@ -64,7 +62,7 @@ public TmResponse act(final Update update) throws IOException {
                 ntf -> Collections.singleton(
                     new MapEntry<>(
                         ntf.subject().getString("title"),
-                        new FormattedText("click:notification#%s", ntf.tid()).toString()
+                        new FormattedText("click:notification#%s", ntf.tid()).asString()
```

Comment:
@olenagerasimova good find, for the record this change will become useless once #40 is solved, but I prefer the usage of `asString()` like you did when possible :)

Reply:
@victornoel I had to correct it, test can not work otherwise

Reply:
@olenagerasimova I know, I was just sharing information :)